src/animation/PropertyBinding.js

```diff
@@ -111,33 +114,41 @@ Object.assign( PropertyBinding, {
 	 */
 	sanitizeNodeName: function ( name ) {
 
-		return name.replace( /\s/g, '_' ).replace( /[^\w-]/g, '' );
+		var reservedRe = new RegExp( '[' + RESERVED_CHARS_RE + ']', 'g' );
+
+		return name.replace( /\s/g, '_' ).replace( reservedRe, '' );
```

Comment:
How about making `reservedRe` closure variable not to execute `new RegExp` for each call?

    sanitizeNodeName: function () {

        var reservedRe = new RegExp( '[' + RESERVED_CHARS_RE + ']', 'g' );

        return function sanitizeNodeName( name ) {

            return name.replace( /\s/g, '_' ).replace( reservedRe, '' );

        };

    }(),

Reply:
Done 👍